specification/json/a2a.json

```diff
@@ -1276,18 +1280,18 @@
             "description": "Configuration details for a supported OAuth Flow",
             "properties": {
                 "refreshUrl": {
-                    "description": "The URL to be used for obtaining refresh tokens. This MUST be in the form of a URL. The OAuth2\nstandard requires the use of TLS.",
+                    "description": "The URL to be used for obtaining refresh tokens. This MUST be in the form of a URL. The OAuth2 \nstandard requires the use of TLS.",
                     "type": "string"
                 },
                 "scopes": {
                     "additionalProperties": {
                         "type": "string"
                     },
-                    "description": "The available scopes for the OAuth2 security scheme. A map between the scope name and a short\ndescription for it. The map MAY be empty.",
+                    "description": "The available scopes for the OAuth2 security scheme. A map between the scope name and a short \ndescription for it. The map MAY be empty.",
                     "type": "object"
                 },
                 "tokenUrl": {
-                    "description": "The token URL to be used for this flow. This MUST be in the form of a URL. The OAuth2 standard\nrequires the use of TLS.",
+                    "description": "The token URL to be used for this flow. This MUST be in the form of a URL. The OAuth2 standard \nrequires the use of TLS.",
```

Comment:
I excluded this generated file from spelling/linting, so these errors shouldn't show up.